plugins/kpexec.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: kpexec
+spec:
+  homepage: https://github.com/ssup2/kpexec
+  shortDescription: Execute process with privilege in a pod
+  description: Execute process with privilege in a pod
```

Comment:
Please go into detail of how it works here (80 character wrap multi line string)